### PR TITLE
Main Header: Background using css

### PR DESCRIPTION
**Pull request by AlexMunoz**
# Details

## Description

remove the div and .png file and implement the background using css. 

## Issue

- #98

### CODE REVIEW

**Comment by FacundoGFlores** [comment]
Good job. I would rather suggest to move the colors to `_variables.scss`. The palette should define a `$white` and also the greens in the background as well.

File: src/pages/home/assets/home.scss [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-
-.mainHeaderBackground {
-  background: url('./main-header-background.png') no-repeat;
-  height: 100%;
-  width: 100%;
-  background-size: cover;
-  position: absolute;
-  z-index: -1; 
+  min-height: 100vh;
+  background-color: #6dd4b1;
```
Comment (crysfel) [inline src/pages/home/assets/home.scss]: Please use the colors defined in `src/theme/base/variables.scss`.

**Comment by emmawedekind** [comment]
I'm sorry but I'm confused by all the comments on these issues.. can you please explain?

**Comment by AlexMunoz** [comment]
i mean, #73 made better changes in the home layout to follow the design, this solution fix a the previous  state. I think it's better to apply #73 and then fix the issue with the background.

**Comment by palashmon** [comment]
I think we should use SVG here also instead of PNG for `/assets/main-header-background.png` based on same critria mentioned in issue #100


> The home page is displaying an illustration in the right side, currently is a `assets/logo.png` image, however we need to use a SVG instead of a png file. The reason for this is because we want to display nice and crispy images in retina/large displays.

**Comment by tureey** [comment]
Good point @AlexMunoz ! I think the same.

**Comment by crysfel** [comment]
@AlexMunoz can you resolve the conflicts and update the code based on the comments? PR #73 has been merged!

**Comment by AlexMunoz** [comment]
i fixed my commit

**Pull request merged**